src/prepack-cli.js

```diff
@@ -56,7 +56,7 @@ function run(
     --trace                  Traces the order of module initialization.
     --serialize              Serializes the partially evaluated global environment as a program that recreates it.
                              (default = true)
-    --check                  Check whole program for diagnostic messages. Do not serialize or produce residual code.
```

Comment:
Make it more clear that this is not an option that would help an end-user in any way.